src/views/AddLiquidity/index.tsx

```diff
@@ -1,7 +1,7 @@
 import { useCallback, useEffect, useState } from 'react'
 import { BigNumber } from '@ethersproject/bignumber'
 import { TransactionResponse } from '@ethersproject/providers'
-import { Currency, currencyEquals, ETHER, TokenAmount, WETH } from '@pancakeswap/sdk'
+import { Currency, currencyEquals, ETHER, TokenAmount, WETH, JSBI } from '@pancakeswap/sdk'
```

Comment:
Could you help me fix eslint on this PR? Thanks

Reply:
Done